
2024-04-25 14:26:02.038015: I external/local_tsl/tsl/platform/default/subprocess.cc:304] Start cannot spawn child process: No such file or directory
2024-04-25 14:26:06.903345: W tensorflow/core/common_runtime/type_inference.cc:339] Type inference failed. This indicates an invalid graph that escaped type checking. Error message: INVALID_ARGUMENT: expected compatible input types, but input 1:
type_id: TFT_OPTIONAL
args {
  type_id: TFT_PRODUCT
  args {
    type_id: TFT_TENSOR
    args {
      type_id: TFT_INT32
    }
  }
}
 is neither a subtype nor a supertype of the combined inputs preceding it:
type_id: TFT_OPTIONAL
args {
  type_id: TFT_PRODUCT
  args {
    type_id: TFT_TENSOR
    args {
      type_id: TFT_FLOAT
    }
  }
}
	for Tuple type infernce function 0
	while inferring type of node 'cond_36/output/_23'
2024-04-25 14:26:07.793083: I external/local_xla/xla/service/service.cc:168] XLA service 0x7f85e277e5e0 initialized for platform CUDA (this does not guarantee that XLA will be used). Devices:
2024-04-25 14:26:07.793132: I external/local_xla/xla/service/service.cc:176]   StreamExecutor device (0): NVIDIA GeForce RTX 4070 Laptop GPU, Compute Capability 8.9
2024-04-25 14:26:07.806124: I external/local_xla/xla/stream_executor/cuda/cuda_dnn.cc:454] Loaded cuDNN version 8907
WARNING: All log messages before absl::InitializeLog() is called are written to STDERR
I0000 00:00:1714047967.848024  617205 device_compiler.h:186] Compiled cluster using XLA!  This line is logged at most once for the lifetime of the process.











